### PR TITLE
Add glm

### PR DESCRIPTION
**Pull request by mcs07**

### CODE REVIEW

**Comment by mcs07** [comment]
The linux build on CircleCI seems to have failed with an error related to CircleCI infrastructure, is there a way to re-trigger it?

**Comment by jakirkham** [comment]
Restarted :)

File: recipes/glm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  url: https://github.com/g-truc/glm/releases/download/{{ version }}/glm-{{ version }}.zip
+  sha256: 209b5943d393925e1a6ecb6734e7507b8f6add25e72a605b25d0d0d382e64fd4
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
```
Comment (hmaarrfk) [inline recipes/glm/meta.yaml]: I've been going around adding `- make   # [unix]` as a build dependency for many packages. Not really important now though.

**Comment by hmaarrfk** [comment]
@conda-forge/staged-recipes I think this is ready

**Pull request merged**